app/screens/ExploreScreen/ExploreScreen.tsx

```diff
@@ -102,6 +105,19 @@ export const ExploreScreen: FC<TabScreenProps<"Explore">> = () => {
       )}
       {!isLoading && sections.length > 0 && (
         <SectionList
+          ListFooterComponent={
+            <ButtonLink
+              openLink={() =>
+                openLinkInBrowser(
```

Comment:
Opening this link doesn't appear to work on android both on device and in sim.

Reply:
fixed here: https://github.com/infinitered/ChainReactApp2023/pull/150/commits/abb1d403c239d30a3d99d3f2833ad46ff9bbed63